lib/react.dart

```diff
@@ -328,11 +328,12 @@ abstract class Component {
   bool shouldComponentUpdate(Map nextProps, Map nextState) => true;
 
   /// > __DEPRECATED - DO NOT USE__
-  /// >
+  ///
```

Comment:
Why were these removed?

Reply:
No deliberate reason - just didn't think to add them on the empty lines when I pulled the comment back in. I updated it!